Header header
float32 distance
string id
stroll_bearnav/Feature[] feature
